packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -243,6 +244,7 @@ export class Legend extends BaseProperties {
     private readonly proxyLegendToolbar: HTMLDivElement;
     private readonly proxyLegendPagination: HTMLDivElement;
     private readonly proxyLegendItemDescription: HTMLParagraphElement;
+    private readonly proxyLegendToolbarDestroyFns: DestroyFns = new DestroyFns();
```

Comment:
Note: I suspect that this is what's causing the memory benchmark failure. When there are a lot of legend item, this array will be very big because they are many event listeners to remove.

Reply:
Obsolete. CI now passes.